README.md

```diff
@@ -74,6 +74,14 @@ The names `"semi"` and `"quotes"` are the names of [rules](https://eslint.org/do
 
 The three error levels allow you fine-grained control over how ESLint applies rules (for more configuration options and details, see the [configuration docs](https://eslint.org/docs/latest/use/configure)).
 
+## Version Support
+
+We will continue to release bug and security fixes for versions 8 and 9. The prvious versions are end of life and will not receive any security or bug fixes.
+
+See our [`Version Support](https://eslint.org/version-support/) document for more information
```

Comment:
Maybe we should make these urls a relative one instead. so even in other deployments we can resolve to that particular version instead of prod one.